lib/vdsm/storage/blockVolume.py

```diff
@@ -164,7 +164,16 @@ def getParent(self):
         """
         Return parent volume UUID
         """
-        return self.getParentTag()
+        parent_tag = self.getParentTag()
+        parent_meta = self.getParentMeta()
+        if parent_tag != parent_meta:
+            self.log.warning(
+                "volUUID=%s parent in tag (%s) and metadata (%s) differ",
+                self.volUUID, parent_tag, parent_meta)
```

Comment:
Logging a warning is good and we can merge this now quickly. This will give us easy
way to understand how common is this issue. If you split the change to simple PR adding the warning we can merge it quickly.